packages/app/public/icons.svg

```diff
@@ -354,5 +354,19 @@
         stroke-linejoin="round"
       />
     </symbol>
+    <symbol id="openeye" viewBox="0 0 24 24" fill="none">
```

Comment:
Can you use the eye icons from Figma? https://www.figma.com/file/Q79i1WVA1bIAzYaLihgteN/Snort?node-id=68-4951&t=DDTOEgw4924vgYGt-0

Reply:
@v0l I dont have access to download icons from here..  Can you help me with the icons please? Or, any way to download is also helpful.

Reply:
If you just sign into Figma it will let you, you just need any account

Reply:
Changed the icons to the ones from Figma